DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -88,7 +88,7 @@ def run_connect(self, device, command, *argv, **kwarg):
             IxnetworkIxiaClientImpl.tis = []
             crc = IxnetworkIxiaClientImpl.ixnet.Traffic.TrafficItem.ConfigElement._SDM_ENUM_MAP["crc"]
             IxnetworkIxiaClientImpl.bad_crc = {True: crc[0], False: crc[1]}
-
+            IxnetworkIxiaClientImpl.ixnet.Traffic.EnableMinFrameSize = True
```

Comment:
It would be good to be able to enable/disable this option. As I understand reducing the signature length may affect some tracking features. We should not decrease TGen functionality by doing this by default.